README.md

```diff
@@ -151,13 +151,13 @@ everything else from the plugin:
 <Chain "FilterOutDetailedDockerStats">
   <Rule "CpuUsage">
```

Comment:
if this rule is changed to work only with cpu.percent then the rule name should probably change to CpuPercent.